### PR TITLE
[PAN-3229] Add `--required-block` to list of CLI options

### PR DESCRIPTION
**Pull request by sgregglives**
Signed-off-by: Steven Gregg <steven.gregg@consensys.net>

## PR description
Update to the CLI options to include `--required-block`
## Fixed Issue(s)

PAN-3229

### CODE REVIEW

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -1134,6 +1134,26 @@ remote-connections-max-percentage=25
...
+
+```bash tab="Environment Variable"
+BESU_REQUIRED_BLOCK=6485846=0x43f0cd1e5b1f9c4d5cda26c240b59ee4f1b510d0a185aa8fd476d091b0097a80
+```
+
+```bash tab="Configuration File"
+required-block=6485846=0x43f0cd1e5b1f9c4d5cda26c240b59ee4f1b510d0a185aa8fd476d091b0097a80
+```
+
+Block number and hash required by peers.
```
Comment (MadelineMurray) [inline docs/Reference/CLI/CLI-Syntax.md]: This needs a bit more information - required by peers to what?

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -1134,6 +1134,26 @@ remote-connections-max-percentage=25
...
+```bash tab="Command Line"
+--required-block=6485846=0x43f0cd1e5b1f9c4d5cda26c240b59ee4f1b510d0a185aa8fd476d091b0097a80
+```
+
+```bash tab="Environment Variable"
+BESU_REQUIRED_BLOCK=6485846=0x43f0cd1e5b1f9c4d5cda26c240b59ee4f1b510d0a185aa8fd476d091b0097a80
+```
+
+```bash tab="Configuration File"
+required-block=6485846=0x43f0cd1e5b1f9c4d5cda26c240b59ee4f1b510d0a185aa8fd476d091b0097a80
```
Comment (MadelineMurray) [inline docs/Reference/CLI/CLI-Syntax.md]: Doesn't either the environment variable or the configuration file require quotes around it?

Comment (shemnon) [reply docs/Reference/CLI/CLI-Syntax.md]: Yes, TOML needs quotes.

File: docs/Reference/CLI/CLI-Syntax.md (hunk 1 above)
Comment (shemnon) [reply docs/Reference/CLI/CLI-Syntax.md]:
```suggestion
Requires a peer with the block number to have a specific hash when connecting or that peer will be rejected.
```

Comment (sgregglives) [reply docs/Reference/CLI/CLI-Syntax.md]: Updated this description with a slight modification

**Pull request merged**